component-catalog-connectors/artifactory-connector/artifactory_catalog_connector/artifactory-catalog.json

```
@@ -0,0 +1,172 @@
+{
+  "$schema": "https://raw.githubusercontent.com/elyra-ai/elyra/master/elyra/metadata/schemas/meta-schema.json",
```

On 6/6 we've renamed the `master` branch to `main` (https://github.com/elyra-ai/elyra/discussions/2772)

```suggestion
  "$schema": "https://raw.githubusercontent.com/elyra-ai/elyra/main/elyra/metadata/schemas/meta-schema.json",
```

---

Resolved in https://github.com/elyra-ai/examples/pull/99/commits/b97b758ec6867f83c646bb560d12cfb80f7ee7bb